upgrade/util/util.go

```diff
@@ -149,6 +159,97 @@ func CreateOrUpdateLonghornVersionSetting(namespace string, lhClient *lhclientse
 	return nil
 }
 
+// IsUpgradePathSupported returns if the upgrade path from lhVersionBeforeUpgrade to meta.Version is supported.
+func IsUpgradePathSupported(namespace string, lhClient *lhclientset.Clientset) bool {
+	lhVersionBeforeUpgrade, err := GetCurrentLonghornVersion(namespace, lhClient)
```

Comment:
Can we add a unit test for this? so we can test the upgrade path combination. 

As I mentioned in another comment, make this function return error with message instead. 

Also, this is an exported function, so besides adding the brief intro, I would suggest briefly explaining what paths will be supported and validated.

Reply:
Added a unit test, and added the brief introduction for the function.
Thanks.